k8s/prod/common/family-public-law/fpl-case-service.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: fpl-case-service
+  namespace: family-public-law
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: fpl-case-service
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: fpl-case-service
+    version: 1.1.0
+  values:
+    java:
+      replicas: 2
+      memoryRequests: '512Mi'
+      cpuRequests: '250m'
+      memoryLimits: '2048Mi'
+      cpuLimits: '1000m'
+      readinessDelay: 45
+      readinessTimeout: 5
+      readinessPeriod: 15
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: 1dfa1cca-46c8-4f9a-a4d9-adea4a7f3ed6
+      image: hmctspublic.azurecr.io/fpl/case-service:prod-493b765c
+      ingressHost: fpl-case-service-prod.service.core-compute-prod.internal
+      environment:
+        JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MinRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
+          -XX:+UseConcMarkSweepGC -javaagent:/opt/app/applicationinsights-agent-2.5.0-BETA.3.jar
+        IDAM_API_URL: https://idam-api.platform.hmcts.net
+        IDAM_S2S_AUTH_URL: http://rpe-service-auth-provider-prod.service.core-compute-prod.internal
+        CORE_CASE_DATA_API_URL: http://ccd-data-store-api-prod.service.core-compute-prod.internal
+        DOCUMENT_MANAGEMENT_URL: http://dm-store-prod.service.core-compute-prod.internal
+        CCD_UI_BASE_URL: https://www.ccd.platform.hmcts.net
+      aadIdentityName: family-public-law
+      keyVaults:
+        fpl:
```

Comment:
shouldn't this be in double quotes?

Reply:
This is simple value, no need for quotes IMO. Same works just fine in other envs.

Reply:
doesn't matter afaik